sigstore-java/src/test/java/dev/sigstore/encryption/KeysTest.java

```diff
@@ -184,6 +185,13 @@ void parseTufPublicKey_rsa()
     }
   }
 
+  @Test
+  void parsePublicKeyBad()
```

Comment:
can you update naming here `parsePublicKey_failOnNullSection()`. If you could also point to the oss_fuzz failure in a comment, I think that makes this a bit easier to track to origin since the testcase data is a bit weird.

Reply:
Change accordingly, added a link to the oss-fuzz bug page in the comment.